Documentation/client.md

```diff
@@ -106,31 +106,31 @@ the s3cmd tool is included in the [Rook toolbox](toolbox.md) pod.
 1. Get the connection information for accessing object storage
 
    ```bash
-   eval $(rookctl object connection mystore rook-user --format env-var)
+   eval $(rookctl object connection my-store rook-user --format env-var)
```

Comment:
I think this is done with https://github.com/rook/rook/pull/1246.

Reply:
that was a different doc change actually